test/systemtests/bgp_test.go

```diff
@@ -510,19 +508,15 @@ func (s *systemtestSuite) TestBgpTriggerNetpluginRestart(c *C) {
 	for _, node := range s.nodes {
 		var err error
 		c.Assert(node.stopNetplugin(), IsNil)
-		logrus.Info("Sleeping for a while to wait for netplugin's TTLs to expire")
-		time.Sleep(1 * time.Minute)
```

Comment:
This requires some checks to ensure whatever is meant to expire has expired. We can't remove these two sleep calls without ensuring the relevant TTL has expired. The BGP tests run into some failures right now.